### DISCUSSION TITLE
migrate data between 2 kimai2 installations

### DISCUSSION THREAD

**Discussion by VOOM108** [kimai/kimai#2633] [On-Premise: Installation & Updates]
Is it possible? Like migrating v1 data to kimai2? 

I have been stuck at kimai2 1.1 until tonight. The installation is half broken but still did, what I needed it to. I was not able to update the installation, so I made a second install with the current version. Now it would be great to be able to migrate customers, projects etc. into the clean install.

**Comment by kevinpapst** [comment]
Yes. Same as a backup restore: https://www.kimai.org/documentation/backups.html

**Reply by kevinpapst** [reply]
Did it work @VOOM108 ?

**Reply by VOOM108** [reply]
I never tried, I started everything new starting July, as my whole structure of clients, projects etc. was not really making sense any more. Was long overdue anyway.

**Reply by kevinpapst** [reply]
Okay, avoiding work is also a solution 😁
